{"name": "test_active_when_lock", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 0\n  -1\n  +0", "trace": "self = <test_ui.Test_UI_Auth object at 0x000001C1EB846280>\nspecial_user = ('test_na_na', '123456')\nsql_client = <sql_client.client.MysqlOrmConnection object at 0x000001C1EB9A9100>\nnorm_auth = <api_client.api_client.Client object at 0x000001C1EB9A9BE0>\n\n    @allure.story('other')\n    @pytest.mark.other\n    def test_active_when_lock(self, special_user, sql_client, norm_auth):\n        \"\"\"Проверка изменится ли active при блокировке пользователя\"\"\"\n        norm_auth.unlock(special_user[0])\n        self.page.auth(*special_user)\n        norm_auth.lock(special_user[0])\n>       assert sql_client.select('username', special_user[0])[0][5] == 0\nE       assert 1 == 0\nE         -1\nE         +0\n\ntests\\test_ui.py:102: AssertionError"}, "description": "Проверка изменится ли active при блокировке пользователя", "start": 1590592399903, "stop": 1590592407852, "uuid": "aeef09c8-359d-4f1d-abdb-01bb5457910d", "historyId": "ec871b40af13902936ea734e08206ec4", "testCaseId": "928c1c598e6dc79ea9f9da88ace276b7", "fullName": "tests.test_ui.Test_UI_Auth#test_active_when_lock", "labels": [{"name": "story", "value": "other"}, {"name": "tag", "value": "other"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "Test_UI_Auth"}, {"name": "host", "value": "DESKTOP-4CMVFT5"}, {"name": "thread", "value": "14640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}